### PR TITLE
[User] #685 Add Remote access option (api_key_enable, true by default) to user form

### PR DESCRIPTION
**Pull request by crouillon**

### CODE REVIEW

File: src/tb/apps/user/entity/user.js [hunk 1]
```diff
@@ -73,6 +74,10 @@ define(['Core/Utils', 'cryptojs.md5', 'jsclass'], function (Utils, CryptoJS) {
             return (this.data.activated !== undefined && this.data.activated !== false) ? true : false;
         },
 
+        api_key_enabled: function () {
```
Comment (fkroockmann) [inline src/tb/apps/user/entity/user.js]: Can you use CamelCase syntax ?

**Comment by fkroockmann** [comment]
:+1:

File: src/tb/apps/user/entity/user.js (hunk 1 above)
Comment (crouillon) [inline src/tb/apps/user/entity/user.js]: eheh, I really don't know: the API response give api_key_enabled, I though that the function should  match it, is it correct?

Comment (fkroockmann) [inline src/tb/apps/user/entity/user.js]: Mmh ok, you are right :)

**Comment by ndufreche** [comment]
:+1:  thanks

**Pull request merged**